modules/study/src/main/StudySocket.scala

```diff
@@ -124,7 +124,7 @@ final private class StudySocket(
         case "addChapter" =>
           reading[ChapterMaker.Data](o) { data =>
             val sticky = o.obj("d").flatMap(_.boolean("sticky")) | true
-            who foreach api.addChapter(studyId, data, sticky = sticky)
+            who foreach api.addChapter(studyId, data, sticky = sticky, withRatings = true)
```

Comment:
I was a little confused by this file so let me know if I should do something differently here.